prime-router/docs/api/delivery.yml

```diff
@@ -8,6 +8,39 @@ info:
     email: reportstream@cdc.gov
   version: 0.2.0-oas3
 paths:
+  /v1/receivers/{receiverFullName}/deliveries:
```

Comment:
POST parameters do not occur in the path.

This is the first API to use a versioning based path.  Is there an agreement that we will use this format moving forward?  Is there a plan on how to handle the existing API's (add a /v1/ alias)?